languages/julia/config.json

```diff
@@ -29,11 +29,7 @@
                 "name": "The Exercism Matrix",
                 "slug": "exercism-matrix",
                 "concepts": [
-                    "matrices-introduction",
-                    "matrices-concatenation",
-                    "matrices-indices",
-                    "matrices-iteration",
-                    "matrices-mutation"
+                    "matrices-introduction"
```

Comment:
For the record, the `-introduction` suffix goes against the [naming guidelines](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts), but you probably knew that already :)

Reply:
Yes.

Reply:
Can you suggest a compliant name? How about `matrices-basics`?

Or is the idea to have a single `matrices` concept and teach it in multiple exercises?